projects/storefrontlib/cms-components/navigation/scroll-to-top/scroll-to-top.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  CmsConfig,
+  provideDefaultConfig,
+  ScrollBehavior,
+} from '@spartacus/core';
+import { IconModule } from '../../misc/icon/icon.module';
+import { ScrollToTopComponent } from './scroll-to-top.component';
+
+@NgModule({
+  imports: [CommonModule, IconModule],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ScrollToTopComponent: {
+          component: ScrollToTopComponent,
+          data: {
+            behavior: ScrollBehavior.SMOOTH,
+            displayThreshold: 200,
```

Comment:
What is the expected a way for customers to configure the component?

Reply:
We don't know, I guess it was not specified. We had to choose between cms component data or global config. What is your opinion?

Reply:
Not sure, that's why I was asking.  If we don't expect customers to override cms component mappings to dial down these options, we could use the global config.  Is there a convention already in Spartacus for this kind of use case?  I can dig a little bit.

Reply:
We do have a few other components with component specific ui properties in the data field.